src/styles/header.scss

```diff
@@ -0,0 +1,66 @@
+.header {
+  max-width: 1400px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+
+  .logo {
+    display: flex;
+    align-items: center;
+    margin-left: $header-margin;
+  }
+
+  .nav {
+    margin: 0 $header-margin 0 0;
+  }
+
+  .nav ul {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    list-style: none;
+    padding: 0;
+
+    :not(:last-child) {
+      margin-right: 20px;
+    }
```

Comment:
The selector `:not(:last-child)` is incorrectly placed. It should be applied to the `.nav__item` elements to ensure that only the list items, except the last one, have the right margin.